### ISSUE TITLE
Intermittent AuTest failures (thread_N_M tests)

### ISSUE THREAD

**Issue by masaori335** [apache/trafficserver#5913]
Another intermittent failure of AuTest. I saw this several times on ci jobs for PRs. 
```
 Test: thread_2_1: Failed
    File: thread_2_1.test.py
    Directory: /var/jenkins/workspace/autest-github/src/tests/gold_tests/thread_config
   Starting Test thread_2_1 : No issues found - Passed
      Reason: Started!
   Process: ts: Passed
     Setting up : MakeDir - Passed
     Setting up : Copy - Passed
     Setting up : MakeDir - Passed
     Setting up : Chown - Passed
     Setting up : Chown - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : Copy - Passed
     Setting up : MakeDir - Passed
     Setting up : Copy - Passed
     Setting up : MakeDir - Passed
     Setting up : Chown - Passed
     Setting up : Chown - Passed
     Setting up : MakeDir - Passed
     Setting up : Chown - Passed
     Setting up : Chown - Passed
     Setting up : MakeDir - Passed
     Setting up : MakeDir - Passed
     Setting up : Chown - Passed
     Setting up : Chown - Passed
     Setting up : MakeDir - Passed
     Setting up : Chown - Passed
     Setting up : Chown - Passed
     Setting up : Lamda - Passed
     Setting up : Lamda - Passed
     Setting up : Lamda - Passed
     Setting up : Lamda - Passed
     Setting up : CopyAs - Passed
     Test : Checking that ReturnCode == 0 - Passed
        Reason: Returned Value: 0 == 0
     file /var/tmp/ausb-5903.11015/thread_2_1/ts/log/diags.log : diags.log should not contain errors - Passed
        Reason: Contents of /var/tmp/ausb-5903.11015/thread_2_1/ts/log/diags.log excludes expression
     file /var/tmp/ausb-5903.11015/thread_2_1/ts/log/diags.log : diags.log should not contain errors - Passed
        Reason: Contents of /var/tmp/ausb-5903.11015/thread_2_1/ts/log/diags.log excludes expression
     Appending File /var/tmp/ausb-5903.11015/thread_2_1/ts/config/records.config : Writing config file records.config - Passed
        Reason: Success
     Appending File /var/tmp/ausb-5903.11015/thread_2_1/ts/config/remap.config : Appended file /var/tmp/ausb-5903.11015/thread_2_1/ts/config/remap.config - Passed
        Reason: Success
   Process: server: Passed
     Setting up : Lamda - Passed
     Setting up : MakeDir - Passed
     Test : Checking that any test passes - Passed
        Reason: One or more tests passed
     
       Test : Checking that ReturnCode == None - Failed
          Reason: Returned Value 0 != None
       
     
       Test : Checking that ReturnCode == 0 - Passed
          Reason: Returned Value: 0 == 0
       
   Process: Default: Skipped
      Reason: Was not started
   Process: Default: Skipped
      Reason: Was not started
   Run: 0-tr: Passed
     Process: Default: Passed
       Test : Checking that ReturnCode == 0 - Passed
          Reason: Returned Value: 0 == 0
       file /var/tmp/ausb-5903.11015/thread_2_1/_output/0-tr-Default/stream.stderr.txt : Checking that /var/tmp/ausb-5903.11015/thread_2_1/_output/0-tr-Default/stream.stderr.txt matches gold/http_200.gold - Passed
          Reason: Values match
     Process: Default: Passed
       Test : Checking that ReturnCode == 0 - Passed
          Reason: Returned Value: 0 == 0
       file /var/tmp/ausb-5903.11015/thread_2_1/_output/0-tr-Default/stream.stderr.txt : Checking that /var/tmp/ausb-5903.11015/thread_2_1/_output/0-tr-Default/stream.stderr.txt matches gold/http_200.gold - Passed
          Reason: Values match
   Run: 1-tr: Failed
     Starting TestRun 1-tr : No Issues found - Passed
        Reason: Started!
     Process: Default: Failed
       Test : Checking that ReturnCode == 0 - Failed
          Reason: Returned Value 1 != 0
     Process: Default: Failed
       Test : Checking that ReturnCode == 0 - Failed
          Reason: Returned Value 1 != 0
```
https://ci.trafficserver.apache.org/job/autest-github/11015/console

**Comment by masaori335** [comment]
"thread_1_1"  fails sometime too.
```
 Test: thread_1_1: Failed
    File: thread_1_1.test.py
    Directory: /var/jenkins/workspace/autest-github/src/tests/gold_tests/thread_config
   Starting Test thread_1_1 : No issues found - Passed
      Reason: Started!
   Process: ts: Passed
     Setting up : MakeDir - Passed
     Setting up : Copy - Passed
     Setting up : MakeDir - Passed
     Setting up : Chown - Passed
     Setting up : Chown - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : Copy - Passed
     Setting up : MakeDir - Passed
     Setting up : Copy - Passed
     Setting up : MakeDir - Passed
     Setting up : Chown - Passed
     Setting up : Chown - Passed
     Setting up : MakeDir - Passed
     Setting up : Chown - Passed
     Setting up : Chown - Passed
     Setting up : MakeDir - Passed
     Setting up : MakeDir - Passed
     Setting up : Chown - Passed
     Setting up : Chown - Passed
     Setting up : MakeDir - Passed
     Setting up : Chown - Passed
     Setting up : Chown - Passed
     Setting up : Lamda - Passed
     Setting up : Lamda - Passed
     Setting up : Lamda - Passed
     Setting up : Lamda - Passed
     Setting up : CopyAs - Passed
     Test : Checking that ReturnCode == 0 - Passed
        Reason: Returned Value: 0 == 0
     file /var/tmp/ausb-5907.10994/thread_1_1/ts/log/diags.log : diags.log should not contain errors - Passed
        Reason: Contents of /var/tmp/ausb-5907.10994/thread_1_1/ts/log/diags.log excludes expression
     file /var/tmp/ausb-5907.10994/thread_1_1/ts/log/diags.log : diags.log should not contain errors - Passed
        Reason: Contents of /var/tmp/ausb-5907.10994/thread_1_1/ts/log/diags.log excludes expression
     Appending File /var/tmp/ausb-5907.10994/thread_1_1/ts/config/records.config : Writing config file records.config - Passed
        Reason: Success
     Appending File /var/tmp/ausb-5907.10994/thread_1_1/ts/config/remap.config : Appended file /var/tmp/ausb-5907.10994/thread_1_1/ts/config/remap.config - Passed
        Reason: Success
   Process: server: Passed
     Setting up : Lamda - Passed
     Setting up : MakeDir - Passed
     Test : Checking that any test passes - Passed
        Reason: One or more tests passed
     
       Test : Checking that ReturnCode == None - Failed
          Reason: Returned Value 0 != None
       
     
       Test : Checking that ReturnCode == 0 - Passed
          Reason: Returned Value: 0 == 0
       
   Process: Default: Skipped
      Reason: Was not started
   Process: Default: Skipped
      Reason: Was not started
   Run: 0-tr: Passed
     Process: Default: Passed
       Test : Checking that ReturnCode == 0 - Passed
          Reason: Returned Value: 0 == 0
       file /var/tmp/ausb-5907.10994/thread_1_1/_output/0-tr-Default/stream.stderr.txt : Checking that /var/tmp/ausb-5907.10994/thread_1_1/_output/0-tr-Default/stream.stderr.txt matches gold/http_200.gold - Passed
          Reason: Values match
     Process: Default: Passed
       Test : Checking that ReturnCode == 0 - Passed
          Reason: Returned Value: 0 == 0
       file /var/tmp/ausb-5907.10994/thread_1_1/_output/0-tr-Default/stream.stderr.txt : Checking that /var/tmp/ausb-5907.10994/thread_1_1/_output/0-tr-Default/stream.stderr.txt matches gold/http_200.gold - Passed
          Reason: Values match
   Run: 1-tr: Failed
     Starting TestRun 1-tr : No Issues found - Passed
        Reason: Started!
     Process: Default: Failed
       Test : Checking that ReturnCode == 0 - Failed
          Reason: Returned Value 1 != 0
     Process: Default: Failed
       Test : Checking that ReturnCode == 0 - Failed
          Reason: Returned Value 1 != 0
```
https://ci.trafficserver.apache.org/job/autest-github/10994/console

**Comment by masaori335** [comment]
thread_2_10 got similar intermittent failure.
https://ci.trafficserver.apache.org/job/autest-github/13379/console
```
 Test: thread_2_10: Failed
    File: thread_2_10.test.py
    Directory: /var/jenkins/workspace/autest-github/src/tests/gold_tests/thread_config
   Starting Test thread_2_10 : No issues found - Passed
      Reason: Started!
   Process: ts: Passed
     Setting up : MakeDir - Passed
     Setting up : Copy - Passed
     Setting up : MakeDir - Passed
     Setting up : Chown - Passed
     Setting up : Chown - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : Copy - Passed
     Setting up : MakeDir - Passed
     Setting up : Copy - Passed
     Setting up : MakeDir - Passed
     Setting up : Chown - Passed
     Setting up : Chown - Passed
     Setting up : MakeDir - Passed
     Setting up : Chown - Passed
     Setting up : Chown - Passed
     Setting up : MakeDir - Passed
     Setting up : MakeDir - Passed
     Setting up : Chown - Passed
     Setting up : Chown - Passed
     Setting up : MakeDir - Passed
     Setting up : Chown - Passed
     Setting up : Chown - Passed
     Setting up : Lamda - Passed
     Setting up : Lamda - Passed
     Setting up : Lamda - Passed
     Setting up : Lamda - Passed
     Setting up : CopyAs - Passed
     Test : Checking that ReturnCode == 0 - Passed
        Reason: Returned Value: 0 == 0
     file /var/tmp/ausb-6844.13379/thread_2_10/ts/log/diags.log : diags.log should not contain errors - Passed
        Reason: Contents of /var/tmp/ausb-6844.13379/thread_2_10/ts/log/diags.log excludes expression
     file /var/tmp/ausb-6844.13379/thread_2_10/ts/log/diags.log : diags.log should not contain errors - Passed
        Reason: Contents of /var/tmp/ausb-6844.13379/thread_2_10/ts/log/diags.log excludes expression
     Appending File /var/tmp/ausb-6844.13379/thread_2_10/ts/config/records.config : Writing config file records.config - Passed
        Reason: Success
     Appending File /var/tmp/ausb-6844.13379/thread_2_10/ts/config/remap.config : Appended file /var/tmp/ausb-6844.13379/thread_2_10/ts/config/remap.config - Passed
        Reason: Success
   Process: server: Passed
     Setting up : Lamda - Passed
     Setting up : MakeDir - Passed
     Test : Checking that any test passes - Passed
        Reason: One or more tests passed
     
       Test : Checking that ReturnCode == None - Failed
          Reason: Returned Value 0 != None
       
     
       Test : Checking that ReturnCode == 0 - Passed
          Reason: Returned Value: 0 == 0
       
   Process: Default: Skipped
      Reason: Was not started
   Process: Default: Skipped
      Reason: Was not started
   Run: 0-tr: Passed
     Process: Default: Passed
       Test : Checking that ReturnCode == 0 - Passed
          Reason: Returned Value: 0 == 0
       file /var/tmp/ausb-6844.13379/thread_2_10/_output/0-tr-Default/stream.stderr.txt : Checking that /var/tmp/ausb-6844.13379/thread_2_10/_output/0-tr-Default/stream.stderr.txt matches gold/http_200.gold - Passed
          Reason: Values match
     Process: Default: Passed
       Test : Checking that ReturnCode == 0 - Passed
          Reason: Returned Value: 0 == 0
       file /var/tmp/ausb-6844.13379/thread_2_10/_output/0-tr-Default/stream.stderr.txt : Checking that /var/tmp/ausb-6844.13379/thread_2_10/_output/0-tr-Default/stream.stderr.txt matches gold/http_200.gold - Passed
          Reason: Values match
   Run: 1-tr: Failed
     Starting TestRun 1-tr : No Issues found - Passed
        Reason: Started!
     Process: Default: Failed
       Test : Checking that ReturnCode == 0 - Failed
          Reason: Returned Value 1 != 0
     Process: Default: Failed
       Test : Checking that ReturnCode == 0 - Failed
          Reason: Returned Value 1 != 0
```

https://ci.trafficserver.apache.org/autest/ausb-6844.13379/thread_2_10/_output/1-tr-Default/stream.all.txt
```
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/psutil/_common.py", line 340, in wrapper
    ret = self._cache[fun]
AttributeError: _cache

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/psutil/_pslinux.py", line 1517, in wrapper
    return fun(self, *args, **kwargs)
  File "/usr/local/lib64/python3.6/site-packages/psutil/_common.py", line 343, in wrapper
    return fun(self)
  File "/usr/local/lib64/python3.6/site-packages/psutil/_pslinux.py", line 1559, in _parse_stat_file
    with open_binary("%s/%s/stat" % (self._procfs_path, self.pid)) as f:
  File "/usr/local/lib64/python3.6/site-packages/psutil/_common.py", line 604, in open_binary
    return open(fname, "rb", **kwargs)
FileNotFoundError: [Errno 2] No such file or directory: '/proc/17568/stat'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/psutil/__init__.py", line 474, in _init
    self.create_time()
  File "/usr/local/lib64/python3.6/site-packages/psutil/__init__.py", line 824, in create_time
    self._create_time = self._proc.create_time()
  File "/usr/local/lib64/python3.6/site-packages/psutil/_pslinux.py", line 1517, in wrapper
    return fun(self, *args, **kwargs)
  File "/usr/local/lib64/python3.6/site-packages/psutil/_pslinux.py", line 1729, in create_time
    ctime = float(self._parse_stat_file()['create_time'])
  File "/usr/local/lib64/python3.6/site-packages/psutil/_pslinux.py", line 1524, in wrapper
    raise NoSuchProcess(self.pid, self._name)
psutil.NoSuchProcess: psutil.NoSuchProcess process no longer exists (pid=17568)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "check_threads.py", line 89, in <module>
    main()
  File "check_threads.py", line 85, in main
    exit(count_threads(args.ts_path, args.etnet_threads, args.accept_threads))
  File "check_threads.py", line 31, in count_threads
    p = psutil.Process(pid)
  File "/usr/local/lib64/python3.6/site-packages/psutil/__init__.py", line 447, in __init__
    self._init(pid)
  File "/usr/local/lib64/python3.6/site-packages/psutil/__init__.py", line 487, in _init
    raise NoSuchProcess(pid, None, msg)
psutil.NoSuchProcess: psutil.NoSuchProcess no process found with pid 17568
```

**Comment by ywkaras** [comment]
thread_1_0 was seen to fail recently.

**Comment by ywkaras** [comment]
I ran the thread_1_0 test 500 times and it failed twice, here is the sandbox:  https://github.com/ywkaras/MiscRepo/blob/master/FILES/thread_0_1_sb.tar.gz .

**Comment by ywkaras** [comment]
I think the above failures may have been because of faulty docker installation.  I will retry.

**Comment by ywkaras** [comment]
Saw a single failure on the retry.  It looks like TS froze very early in the initialization process.  diags.log does not exist.  stream.all.out has zero bytes for the ts process.
```
Generating Report: --------------
 Test: ___tmp___427: Failed
    File: ___tmp___427.test.py
    Directory: /Users/wkaras/REPOS/TS2/tests/gold_tests/thread_config
   Starting Test ___tmp___427 : No issues found - Passed
      Reason: Started!
   Process: ts: Failed
     Setting up : RunCommand - Passed
     Setting up : MakeDir - Passed
     Setting up : MakeDir - Passed
     Setting up : Chown - Passed
     Setting up : Chown - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : CopyAs - Passed
     Setting up : Copy - Failed
        Reason: Cannot copy /Users/wkaras/TSX/TS2/etc/trafficserver/body_factory to /Users/wkaras/REPOS/TS2/tests/_sandbox/___tmp___427/ts/config/body_factory because:
          Cannot copy /Users/wkaras/TSX/TS2/etc/trafficserver/body_factory because [Errno 5] Input/output error
     Setting up : MakeDir - Passed
     Setting up : Copy - Passed
     Setting up : Chown - Passed
     Setting up : Chown - Passed
     Setting up : Chown - Passed
     Setting up : Chown - Passed
     Setting up : MakeDir - Passed
     Setting up : MakeDir - Passed
     Setting up : Chown - Passed
     Setting up : Chown - Passed
     Setting up : MakeDir - Passed
     Setting up : Chown - Passed
     Setting up : Chown - Passed
     Setting up : Lamda - Passed
     Setting up : Lamda - Passed
     Setting up : Lamda - Passed
     Setting up : Lamda - Passed
     Setting up : CopyAs - Passed
     Test : Checking that ReturnCode == 0 - Failed
        Reason: Returned Value None != 0
     file /Users/wkaras/REPOS/TS2/tests/_sandbox/___tmp___427/ts/log/diags.log : diags.log should not contain errors - Failed
        Reason: Cannot read /Users/wkaras/REPOS/TS2/tests/_sandbox/___tmp___427/ts/log/diags.log: [Errno 2] No such file or directory: '/Users/wkaras/REPOS/TS2/tests/_sandbox/___tmp___427/ts/log/diags.log'
     file /Users/wkaras/REPOS/TS2/tests/_sandbox/___tmp___427/ts/log/diags.log : diags.log should not contain errors - Failed
        Reason: Cannot read /Users/wkaras/REPOS/TS2/tests/_sandbox/___tmp___427/ts/log/diags.log: [Errno 2] No such file or directory: '/Users/wkaras/REPOS/TS2/tests/_sandbox/___tmp___427/ts/log/diags.log'
   Process: server: Passed
   Run: 0-tr: Failed
     Starting TestRun 0-tr : Waiting for Process "ts" to become ready - Failed
        Reason: Process finished before it was ready
     Process: Default: Skipped
        Reason: Was not started
   Run: 1-tr: Failed
     Starting TestRun 1-tr : No Issues found - Passed
        Reason: Started!
     Process: Default: Failed
       Test : Checking that ReturnCode == 0 - Failed
          Reason: Returned Value 1 != 0

Total of 500 test
  Unknown: 0
  Exception: 0
  Failed: 1
  Warning: 0
  Skipped: 0
  Passed: 499
```

**Comment by ywkaras** [comment]
Saw thread_1_0.test.py fail in PR check autest.  (The PR, https://github.com/apache/trafficserver/pull/7232/files , only changed a different Au test.)  The failure happened running the check_threads.py script in the second test run.  Here is the resulting stream.all.txt:
```
Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/psutil/_common.py", line 340, in wrapper
    ret = self._cache[fun]
AttributeError: _cache

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/psutil/_pslinux.py", line 1517, in wrapper
    return fun(self, *args, **kwargs)
  File "/usr/local/lib64/python3.6/site-packages/psutil/_common.py", line 343, in wrapper
    return fun(self)
  File "/usr/local/lib64/python3.6/site-packages/psutil/_pslinux.py", line 1559, in _parse_stat_file
    with open_binary("%s/%s/stat" % (self._procfs_path, self.pid)) as f:
  File "/usr/local/lib64/python3.6/site-packages/psutil/_common.py", line 604, in open_binary
    return open(fname, "rb", **kwargs)
FileNotFoundError: [Errno 2] No such file or directory: '/proc/32116/stat'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib64/python3.6/site-packages/psutil/__init__.py", line 474, in _init
    self.create_time()
  File "/usr/local/lib64/python3.6/site-packages/psutil/__init__.py", line 824, in create_time
    self._create_time = self._proc.create_time()
  File "/usr/local/lib64/python3.6/site-packages/psutil/_pslinux.py", line 1517, in wrapper
    return fun(self, *args, **kwargs)
  File "/usr/local/lib64/python3.6/site-packages/psutil/_pslinux.py", line 1729, in create_time
    ctime = float(self._parse_stat_file()['create_time'])
  File "/usr/local/lib64/python3.6/site-packages/psutil/_pslinux.py", line 1524, in wrapper
    raise NoSuchProcess(self.pid, self._name)
psutil.NoSuchProcess: psutil.NoSuchProcess process no longer exists (pid=32116)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "check_threads.py", line 91, in <module>
    main()
  File "check_threads.py", line 87, in main
    exit(count_threads(args.ts_path, args.etnet_threads, args.accept_threads))
  File "check_threads.py", line 31, in count_threads
    p = psutil.Process(pid)
  File "/usr/local/lib64/python3.6/site-packages/psutil/__init__.py", line 447, in __init__
    self._init(pid)
  File "/usr/local/lib64/python3.6/site-packages/psutil/__init__.py", line 487, in _init
    raise NoSuchProcess(pid, None, msg)
psutil.NoSuchProcess: psutil.NoSuchProcess no process found with pid 32116
```

**Comment by ywkaras** [comment]
Here is the diags.log for the failed test above:
```
[Oct  1 12:58:33.158] traffic_server STATUS: opened /var/tmp/ausb-7232.14203/thread_1_0/ts/log/diags.log
[Oct  1 12:58:33.159] traffic_server NOTE: updated diags config
[Oct  1 12:58:33.167] traffic_server WARNING: connection throttle too high, 30000 (throttle) + 192 (internal use) > 4096 (file descriptor limit), using throttle of 3904
[Oct  1 12:58:33.168] traffic_server NOTE: storage.config loading ...
[Oct  1 12:58:33.168] traffic_server NOTE: storage.config finished loading
[Oct  1 12:58:33.170] traffic_server WARNING: ControlMatcher - Cannot open config file: /var/tmp/ausb-7232.14203/thread_1_0/ts/config/cache.config - No such file or directory
[Oct  1 12:58:33.170] traffic_server NOTE: ip_allow.yaml loading ...
[Oct  1 12:58:33.174] traffic_server NOTE: ip_allow.yaml finished loading
[Oct  1 12:58:33.174] traffic_server NOTE: parent.config loading ...
[Oct  1 12:58:33.174] traffic_server WARNING: ControlMatcher - Cannot open config file: /var/tmp/ausb-7232.14203/thread_1_0/ts/config/parent.config - No such file or directory
[Oct  1 12:58:33.174] traffic_server NOTE: parent.config finished loading
[Oct  1 12:58:33.177] traffic_server WARNING: logging configuration '/var/tmp/ausb-7232.14203/thread_1_0/ts/config/logging.yaml' doesn't exist
[Oct  1 12:58:33.184] traffic_server NOTE: logging initialized[3], logging_mode = 3
[Oct  1 12:58:33.184] traffic_server NOTE: Initialized plugin_dynamic_reload_mode: 1
[Oct  1 12:58:33.184] traffic_server NOTE: plugin.config loading ...
[Oct  1 12:58:33.184] traffic_server WARNING: plugin.config failed to load: 2, No such file or directory
[Oct  1 12:58:33.188] traffic_server NOTE: /var/tmp/ausb-7232.14203/thread_1_0/ts/config/sni.yaml loading ...
[Oct  1 12:58:33.188] traffic_server NOTE: /var/tmp/ausb-7232.14203/thread_1_0/ts/config/sni.yaml failed to load
[Oct  1 12:58:33.188] traffic_server WARNING: Loading SNI configuration - filename: /var/tmp/ausb-7232.14203/thread_1_0/ts/config/sni.yaml doesn't exist
[Oct  1 12:58:33.188] traffic_server NOTE: ssl_multicert.config loading ...
[Oct  1 12:58:33.188] traffic_server WARNING: Cannot open SSL certificate configuration from /var/tmp/ausb-7232.14203/thread_1_0/ts/config/ssl_multicert.config - No such file or directory
[Oct  1 12:58:33.189] traffic_server NOTE: /var/tmp/ausb-7232.14203/thread_1_0/ts/config/ssl_multicert.config finished loading
[Oct  1 12:58:33.190] traffic_server NOTE: ssl_multicert.config loading ...
[Oct  1 12:58:33.190] traffic_server WARNING: Cannot open SSL certificate configuration from /var/tmp/ausb-7232.14203/thread_1_0/ts/config/ssl_multicert.config - No such file or directory
[Oct  1 12:58:33.190] traffic_server NOTE: volume.config loading ...
[Oct  1 12:58:33.190] traffic_server WARNING: Cannot open the config file: /var/tmp/ausb-7232.14203/thread_1_0/ts/config/volume.config - No such file or directory
[Oct  1 12:58:33.190] traffic_server NOTE: volume.config finished loading
[Oct  1 12:58:33.200] traffic_server NOTE: remap.config loading ...
[Oct  1 12:58:33.200] traffic_server NOTE: strategies.yaml loading ...
[Oct  1 12:58:33.200] traffic_server NOTE: Unable to stat '/var/tmp/ausb-7232.14203/thread_1_0/ts/config/strategies.yaml': No such file or directory
[Oct  1 12:58:33.202] [ET_NET 0] WARNING: disk header different for disk /var/tmp/ausb-7232.14203/thread_1_0/ts/storage/cache.db: clearing the disk
[Oct  1 12:58:33.205] traffic_server NOTE: remap.config finished loading
[Oct  1 12:58:33.209] [TS_MAIN] NOTE: traffic server running
[Oct  1 12:58:33.219] [ET_NET 0] NOTE: Clearing Disk: /var/tmp/ausb-7232.14203/thread_1_0/ts/storage/cache.db
[Oct  1 12:58:33.221] [ET_NET 0] NOTE: clearing cache directory '/var/tmp/ausb-7232.14203/thread_1_0/ts/storage/cache.db 16384:32766'
[Oct  1 12:58:33.230] [ET_NET 0] WARNING: Cannot open the config file: /var/tmp/ausb-7232.14203/thread_1_0/ts/config/hosting.config - No such file or directory
[Oct  1 12:58:33.230] [ET_NET 0] NOTE: hosting.config loading ...
[Oct  1 12:58:33.231] [ET_NET 0] NOTE: cache enabled
[Oct  1 12:58:33.231] [ET_NET 0] NOTE: Traffic Server is fully initialized.
```

**Comment by bryancall** [comment]
Looks like this has been fixed.  Please reopen if there is still an issue.

**Comment by bneradt** [comment]
For documentation purposes: this was fixed by #7515. Both the thread_config and the thread_N_M tests used that count_threads function which could wind up trying to get information on a closed process.  @duke8253 fixed this issue in #7515 via using the process_iter function.
